_docs/tasks/istio-installation.md

```diff
@@ -32,15 +32,19 @@ git clone https://github.com/istio/istio
 ```bash
 cd istio
 ```
-2. Install the Istio core components (Istio-Manager, Mixer, and Ingress-Controller):
+1. Install the Istio core components (Istio-Manager, Istio-Mixer, and Ingress-Controller):
 ```bash
-kubectl apply -f ./kubernetes/istio.yaml
+kubectl apply -f ./kubernetes/istio-15.yaml # for Kubernetes 1.5
+```
+or
+```bash
+kubectl apply -f ./kubernetes/istio-16.yaml # for Kubernetes 1.6 or later
 ```
 1. Source the Istio configuration file:
 ```bash
 source istio.VERSION
 ```
-3. Download one of the [`istioctl`]({{site.bareurl}}/docs/reference/istioctl.html) client binaries corresponding to your OS: `istioctl-osx`, `istioctl-win.exe`,
+1. Download one of the [`istioctl`]({{site.bareurl}}/docs/reference/istioctl.html) client binaries corresponding to your OS: `istioctl-osx`, `istioctl-win.exe`,
```

Comment:
mtail just committed some changes in install related to site.bareurl and buller points. You can incorporate the same.